File: src/main/java/org/goafabric/jobrunr/job/easy/SimpleJob.java
package org.goafabric.jobrunr.job.easy;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.stereotype.Component;

@Component
public class SimpleJob {
    private final Logger log = LoggerFactory.getLogger(this.getClass());

    public void run() {
        log.info("Up & Running from a background Job");
    }

}

File: src/main/kotlin/org/goafabric/jobrunr/job/easy/SimpleJob.kt
package org.goafabric.jobrunr.job.easy

import org.slf4j.LoggerFactory
import org.springframework.stereotype.Component

@Component
class SimpleJob {
    private val log = LoggerFactory.getLogger(this.javaClass)
    fun run() {
        log.info("Up & Running from a background Job")
    }
}
